packages/block-editor/src/components/block-popover/index.js

```diff
@@ -47,8 +52,34 @@ function BlockPopover(
 		};
 	}, [ selectedElement, lastSelectedElement, __unstableRefreshSize ] );
 
+	const [ popoverAnchorRecomputeCounter, forceRecomputePopoverAnchor ] =
+		useReducer( ( s ) => s + 1, 0 );
```

Comment:
I guess it's unlikely, but I'm always concerned about the integer overflowing with this kind of code (`Number.MAX_VALUE === Number.MAX_VALUE + 1`).

Would a boolean that flips work just as well? Or perhaps using a modulo so that it loops back to zero when it reaches a particular number (`Number.MAX_SAFE_INTEGER`).

Reply:
Good point! I've done that in https://github.com/WordPress/gutenberg/pull/44301/commits/5b72aeefb53432db79100183976da5e9ee3752c1